.github/workflows/build.yml

```diff
@@ -0,0 +1,97 @@
+name: build
+
+on:
+  push:
+    branches:
+      - master
+      - 0.10
+      - travis
```

Comment:
Not sure if `0.10` and `travis` is required here ?

Reply:
I left them in because they existed in the old travis setup. I assume that the travis branch is here so you have a dummy branch you can push CI to without worrying about force pushes breaking stuff for people, and the old 0.9 branch was probably a hold over from when development wasn't done directly on master. Given the fact that development is done directly on master these days I might as well remove it.